docs/integrations/tax.mdx

```diff
@@ -106,22 +106,27 @@ You will be ready to set up a connection with the Tax Provider API when all of t
 * During the [OAuth flow](#installing-the-app), the app requests that the merchant authorize read and write permissions on the Information and Settings API scopes.
 * During the [OAuth flow](#installing-the-app), you received and stored the `client_id` and `access_token`.
 
-For context, the [Tax Provider API Connection endpoints](/docs/rest-contracts/tax-app-connection) add a layer of security for your app. You can use these basic credentials to authenticate API requests that you receive from stores so that you can reject unauthorized requests.
-
-An account login flow lets you capture basic authentication credentials from merchants. We recommend that you provide a login flow within your app iFrame so that you can choose your own authentication method. You can also provide an account registration flow in your iFrame to help merchants create a new tax provider account. Learn more about [designing the app UI](/docs/integrations/apps/guide/ui).
-
-We recommend calling the [Update Connection](/docs/rest-contracts/tax-app-connection#update-a-connection) endpoint immediately after the merchant installs your app and you authenticate the merchant's account credentials. Otherwise, your tax provider will not be displayed when merchants navigate to the **Settings > Setup > Tax** page in the control panel. You will need to include `store_hash`, `provider_id`, and `X-Auth-Token`(`access_token`) values.
-
 ## Configure tax provider settings in the control panel
```

Comment:
This heading really speaks to activities _after_ connecting. Indeed, I'm not completely sold on breaking-out `Tax Connection Endpoints` from `Establishing a connection`.

That said, if we did want to isolate some of these details, which it appears we do, I'd probably recommend instead creating a 3rd tier section under 2nd tier `Establishing a connection` instead.